main.go

```diff
@@ -202,15 +203,11 @@ func validateAnchorDir(anchorDir string, files []string) (_ string, err error) {
 	return anchorDir, nil
 }
 
-func registerWeb(_ context.Context, app *extkingpin.App) {
-	cmd := app.Command("web", "Tools for generating static HTML website based on https://gohugo.io/ on every PR with preview")
-	genCmd := cmd.Command("gen", "Generate versioned docs")
-
-	_ = genCmd.Arg("files", "Markdown file(s) to process.").Required().ExistingFiles()
-
-	// TODO(bwplotka): Generate versioned docs used for Hugo. Open question: Hugo specific? How to adjust links? Should fmt and this be
-	// the same?
+func registerTransform(_ context.Context, app *extkingpin.App) {
+	cmd := app.Command("transform", "Transform markdown files in various ways. For example pre process markdown files to allow it for use for popular static HTML websites based on markdown source code and front matter options.")
+	cfg := cmd.Flag("config", "Path to the YAML file with spec defined in github.com/bwplotka/mdox/pkg/transform.Config").
```

Comment:
Maybe a section can be added to README about `transform` command, with sample config yaml, and that can be linked instead of struct? 🙂

Reply:
Well, we could use mdox-exec with our tool we plan to build and do this at the point? =D Great idea!

Reply:
Sure!